package.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "dog-statsy",
-  "version": "1.2.1",
+  "version": "1.3.0",
   "repository": "segmentio/dog-statsy",
   "description": "dogstatsd client",
   "keywords": [
     "datadog",
     "dogstatsd",
     "statsd"
   ],
-  "dependencies": {
-    "debug": "^2.6.9",
-    "forward-events": "0.0.1"
-  },
+  "dependencies": {},
   "devDependencies": {
-    "mocha": "*"
+    "ava": "^3.3.0",
```

Comment:
ava is okay.. but I think generally segment is coalescing around jest -- see example template: https://github.com/segmentio/ts-node-service-template/

Reply:
probably would've been an easier conversion from mocha --> jest too.

Reply:
probably time to 'publish' this doc and discuss internally for our team https://paper.dropbox.com/doc/Integrations-JS-Code-Style--Auco1OGKypMy9atBob16ZM~eAg-lQyB0y2Ed7X3b5LgIAY14

Reply:
👍 

Never gets old https://dayssincelastjavascriptframework.com/

Reply:
:) for sure -- at least jest is part of the facebook ecosystem which seems to be getting 'standardize' treatment? eh..